framework/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.7 under development
 -----------------------
 
+- Enh #5469: Add mimetype validation by mask in FileValidator
```

Comment:
Need you handle after the changelog line i.e. `(kirsenn)`. Also changelog line should be ordered by type and number.